File: fortran/program5.f90
program go2
    implicit none
    REAL :: usernum

10  write(*, '(A)', advance='no') 'Enter a number less than 10: '
    read(*, *) usernum

    if (usernum >=10) then
        write(*, '(A)') 'Invalid input (>=10).'
        go to 10
    end if

   write(*, '(A)') 'Number is < 10, good job!'
end program go2

File: rust/program5.rs
use std::io::{self, Write};

fn reverse_string(string: String) -> String {
    return string.chars().rev().collect();
}

fn main() {

    let mut user_string = String::new();
    let mut width = String::new();

    print!("Enter a string: ");
    io::stdout().flush().expect("failed to flush stdout");

    io::stdin()
        .read_line(&mut user_string)
        .expect("failed to read line");
    
    let user_string: String = user_string.trim().parse().expect("invalid input");
    let reversed = reverse_string(user_string);

    print!("Reversed: {}", reversed);
}